fe1-web/ingestion/Configure.ts

```diff
@@ -1,12 +1,17 @@
 import { getStore } from 'store';
 import { getNetworkManager } from 'network';
+import { messageRegistry } from 'App';
 import { handleRpcRequests } from './Handler';
 import { makeMessageStoreWatcher } from './Watcher';
+import { configure } from './handlers';
 
 export function configureIngestion() {
   // setup the handler for incoming messages
   getNetworkManager().setRpcHandler(handleRpcRequests);
 
+  // configure the message registry
+  configure(messageRegistry);
```

Comment:
```suggestion
export function configureIngestion(messageRegistry: MessageRegistry) {
  // configure the message handlers
  configure(messageRegistry);

  // setup the handler for incoming messages
  getNetworkManager().setRpcHandler(handleRpcRequests);
```

This way, we don't accept messages from the network _before_ we have configured the handlers.
It's a very theoretical thing, but it's good to keep it clean :-)

Reply:
It's a nice remark, I didn't think about it. Thanks !